masterlist.yaml

```diff
@@ -797,7 +797,10 @@ plugins:
         content:
           - lang: en
             text: 'Regenerate after adding or removing mods or generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/)'
-
+      - type: say
+        content:
+          - lang: en
+            text: 'Delete before generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/), to avoid cyclic dependency.'
```

Comment:
```suggestion
      - type: say
        content:
          - lang: en
            text: 'Delete before generating [Merged Objects.esp](https://www.nexusmods.com/morrowind/mods/46870/), to avoid cyclic dependency.'

```